perses/tests/testsystems.py

```diff
@@ -273,28 +273,30 @@ class T4LysozymeTestSystem(PersesTestSystem):
     """
     def __init__(self):
         super(T4LysozymeTestSystem, self).__init__()
-        environments = ['explicit-complex', 'explicit-peptide', 'implicit-complex', 'implicit-peptide', 'vacuum-complex', 'vacuum-peptide']
+        environments = ['explicit-complex', 'explicit-receptor', 'implicit-complex', 'implicit-receptor', 'vacuum-complex', 'vacuum-receptor']
 
         # Create a system generator for our desired forcefields.
         from perses.rjmc.topology_proposal import SystemGenerator
+        from pkg_resources import resource_filename
+        gaff_xml_filename = resource_filename('perses', 'data/gaff.xml')
         system_generators = dict()
-        system_generators['explicit'] = SystemGenerator(['amber99sbildn.xml', 'tip3p.xml'],
+        system_generators['explicit'] = SystemGenerator([gaff_xml_filename,'amber99sbildn.xml', 'tip3p.xml'],
             forcefield_kwargs={ 'nonbondedMethod' : app.CutoffPeriodic, 'nonbondedCutoff' : 9.0 * unit.angstrom, 'implicitSolvent' : None, 'constraints' : None },
             use_antechamber=False)
         system_generators['explicit-complex'] = system_generators['explicit']
-        system_generators['explicit-peptide'] = system_generators['explicit']
-        system_generators['implicit'] = SystemGenerator(['amber99sbildn.xml', 'amber99_obc.xml'],
+        system_generators['explicit-receptor'] = system_generators['explicit']
+        system_generators['implicit'] = SystemGenerator([gaff_xml_filename,'amber99sbildn.xml', 'amber99_obc.xml'],
             forcefield_kwargs={ 'nonbondedMethod' : app.NoCutoff, 'implicitSolvent' : app.OBC2, 'constraints' : None },
             use_antechamber=False)
```

Comment:
I think this should be `use_antechamber=True` (default) to have `SystemGenerator` generate parameters on the fly.